lib/inspec/runner.rb

```diff
@@ -134,7 +134,18 @@ def render_output(run_data)
       return if @conf["reporter"].nil?
 
       @conf["reporter"].each do |reporter|
-        result = Inspec::Reporters.render(reporter, run_data)
+        reporter_opts = {}
+        if @conf.key?(:reporter_backtrace_inclusion)
+          reporter_opts[:include_backtrace] = @conf['reporter_backtrace_inclusion']
+        elsif reporter[0] == 'json-automate'
+          reporter_opts[:include_backtrace] = false
+        end
+        if @conf.key?(:reporter_message_truncation)
+          reporter_opts[:message_truncation] = @conf['reporter_message_truncation']
+        elsif reporter[0] == 'json-automate'
+          reporter_opts[:message_truncation] = '200'
```

Comment:
From chatting to @kekaichinose the default for Automate should maybe be to not truncate. On the principle that for compliance, all data should be reported. 
Then Automate users will need to opt in for the reduced report size.